lib/backend/k8s/k8s_test.go

```diff
@@ -491,39 +503,16 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 			Kind:      apiv3.KindNetworkPolicy,
 		}, "")
 		Expect(err).NotTo(HaveOccurred())
-	})
 
-	// Add a defer to wait for policies to clean up.
-	defer func() {
-		log.Warnf("[TEST] Waiting for policies to tear down")
-		It("should clean up all policies", func() {
-			nps := networkingv1.NetworkPolicyList{}
-			err := c.clientSet.NetworkingV1().RESTClient().
-				Get().
-				Resource("networkpolicies").
-				Namespace("default").
-				Timeout(10 * time.Second).
-				Do().Into(&nps)
-			Expect(err).NotTo(HaveOccurred())
-
-			// Loop until no network policies exist.
-			for i := 0; i < 10; i++ {
-				if len(nps.Items) == 0 {
-					return
-				}
-				nps := networkingv1.NetworkPolicyList{}
-				err := c.clientSet.NetworkingV1().RESTClient().
-					Get().
-					Resource("networkpolicies").
-					Namespace("default").
-					Timeout(10 * time.Second).
-					Do().Into(&nps)
-				Expect(err).NotTo(HaveOccurred())
-				time.Sleep(1 * time.Second)
-			}
-			panic(fmt.Sprintf("Failed to clean up policies: %+v", nps))
-		})
-	}()
+		// Delete the object.
+		result := c.clientSet.NetworkingV1().RESTClient().
```

Comment:
bit confused, why do we need this one? Shouldn't it be taken care of in `AfterEach`? I'm sure there's a reason, but im blanking out

Reply:
Yea, should just remove this actually.